integration/duckdb/src/lance/duckdb/video_functions.cc

```diff
@@ -0,0 +1,130 @@
+//  Copyright 2022 Lance Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#include "lance/duckdb/vector_functions.h"
+
+#include <cstdint>
+#include <duckdb/parser/parsed_data/create_scalar_function_info.hpp>
+#include <iostream>
+#include <memory>
+
+namespace lance::duckdb {
+
+template <typename T>
+void L2Distance(::duckdb::DataChunk &args,
```

Comment:
what is the difference of this to https://github.com/eto-ai/lance/blob/main/integration/duckdb/src/lance/duckdb/vector_functions.cc#L25

Reply:
accident